client/Rating&Reviews/PhotoList.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PhotoList = ({photos}) => (
+  <div>
+    {photos.map(photo => <img key={photo.id} className='photo' src={photo.url} alt='product' />)}
+  </div>
+)
+
+PhotoList.propTypes = {
+  photos: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number,
+    url: PropTypes.string
```

Comment:
Here we see an example of prop validation. Arrays need to have their contents specified with `.arrayOf`, and objects need to have their contents specified with `.shape`. Again, this is just to satisfy the linter.

Reply:
Nice execution of shape.